tests/browser-tests.js

```diff
@@ -0,0 +1,57 @@
+/* globals describe, before */
+
+// const expect = require('chai').expect
+const async = require('async')
+const store = require('local-storage-blob-store')
+const tests = require('./repo-test')
+const _ = require('lodash')
+const IPFSRepo = require('../src')
+
+global.Buffer = require('buffer').Buffer
+
+var repoContext = require.context('raw!./test-repo', true)
```

Comment:
Oh wait this is recursively pulling in a folder? sorry didn't realise it, then :+1: